### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dhodovsk**
test

### CODE REVIEW

**Review by TomasTomecek — APPROVED** [review]
Do you want to merge?

**Comment by dhodovsk** [comment]
Not yet please,this could be used for testing ucho-fedmsg.

**Comment by dhodovsk** [comment]
[citest]

**Comment by TomasTomecek** [comment]
Test.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,2 @@
 # rsyslog
 Upstream repository for rsyslog image
-
```
Comment (TomasTomecek) [inline README.md]: test

**Comment by TomasTomecek** [comment]
test

**Comment by TomasTomecek** [comment]
test

**Comment by dhodovsk** [comment]
[citest]

**Comment by dhodovsk** [comment]
[citest]

**Comment by phracek** [comment]
[citest]

**Comment by dhodovsk** [comment]
[citest]

**Comment by dhodovsk** [comment]
[citest]

**Comment by dhodovsk** [comment]
[citest]

**Comment by dhodovsk** [comment]
[citest]

**Pull request closed without merging**